shared/editor/embeds/Kroki.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import Frame from "../components/Frame";
+import Image from "../components/Img";
+import { EmbedProps as Props } from ".";
+
+function Kroki(props: Props) {
+  const embedUrl = props.attrs.matches[0];
+  const params = new URL(embedUrl).pathname.split("/");
+  const titlePrefix = new URL(embedUrl).hostname;
+  const imgGiagram = params[1];
```

Comment:
What is "Giagram"?

Reply:
Kroki has multiple giagram analyses and  picture format ， His basic link is https://kroki.io/<plantuml>/<svg>/**， no title，so  I would like to point out the giagram analyses and  image format

Reply:
Do you mean "diagram"? Or is this some kroki terminology

Reply:
diagram  ，This is their official example ： https://kroki.io/examples.html